platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/internal/cc/impl/ConfigurationCacheProblemReportingIntegrationTest.groovy

```diff
@@ -521,128 +522,58 @@ class ConfigurationCacheProblemReportingIntegrationTest extends AbstractConfigur
         }
     }
 
-    def "task execution problems are reported and fail the build by default invalidating the cache"() {
```

Comment:
ℹ️  replaced by the new tests in `ConfigurationCacheTaskExecutionIntegrationTest`